src/core/cff_parser.js

```diff
@@ -1295,6 +1298,17 @@ var CFFCompiler = (function CFFCompilerClosure() {
       output.add(compiled.output);
       var topDictTracker = compiled.trackers[0];
 
+      // Create names for all of the glyphs so they can be put in the charset
+      // below.
+      let firstGlyphSid = cff.strings.strings.length + NUM_STANDARD_CFF_STRINGS;
```

Comment:
Nit: Assuming that `cff.strings` is an instance of `CFFStrings`, you should be able to use https://github.com/mozilla/pdf.js/blob/df7756d2a4a03831d836977ed47fa9e02f5a19c1/src/core/cff_parser.js#L945-L947 to simply write this as `let firstGlyphSid = cff.strings.count + NUM_STANDARD_CFF_STRINGS;`